contributing.md

```diff
@@ -1,5 +1,22 @@
 Contributing
```

Comment:
Should contributing.md text be moved to the repo readme.md ? -- less button clicks to navigate. Or at least add a blurb about the high fidelity developer fund.

Especially the telegram link for people wanting to know more.

Reply:
We should probably update the Readme to reflect this doc / additional information about the developer fund, but keep this as a separate file too.